### DISCUSSION TITLE
Welcome to Sanchez Discussions!

### DISCUSSION THREAD

**Discussion by nullpainter** [nullpainter/sanchez#56] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by creinemann** [comment]
I am Carl Reinemann, I have the website usradioguy.com . I have used
Sanchez in all sorts of ways to enhance and augment my received GOES and
Himawari imagery, as well as using it to create global geostationary
composites using 5 geo stationary satellites.
